USFMToolsSharp.Renderers.Docx/DocxRenderer.cs

```diff
@@ -643,25 +647,39 @@ public void getRenderedCell(Marker input, StyleConfig config, XWPFTableRow paren
         /// <summary>
         /// Renders a Table of Contents (TOC) in front of the document.
         /// 
-        /// Please set the paragraphs style to "Heading{#}" before 
-        /// invoking this method. Otherwise, it renders an empty TOC.
+        /// Please set the paragraphs style to "Heading{#}". 
+        /// Otherwise, it renders an empty TOC.
         /// </summary>
         private void RenderTOC()
         {
             TOC tocBuilder = new TOC();
             CT_SdtBlock tableOfContents = tocBuilder.Build();
-
-            // add page break after TOC
-            CT_P pBreak = new CT_P();
-            pBreak.AddNewR().AddNewBr().type = ST_BrType.page;
-            
             CT_P pHeader = CreateFrontHeader();
 
-            newDoc.Document.body.Items.Insert(0, tableOfContents);
-            newDoc.Document.body.Items.Insert(1, pBreak);
-            newDoc.Document.body.Items.Insert(2, pHeader);
+            newDoc.Document.body.Items.Add(tableOfContents);
+            newDoc.Document.body.Items.Add(pHeader);
+            newDoc.CreateParagraph().CreateRun().AddBreak(BreakType.PAGE);
 
             newDoc.EnforceUpdateFields();
         }
+
+        private void RenderFrontMatter(USFMDocument frontMatter)
```

Comment:
You don't need to do this, just insert the front matter before the USFM document that you already have. It should work just fine.

Reply:
how could it be in a separate page and not having the header/footer like the body content?

Reply:
also, if the notetaking option is enabled, the license will be indented the same as the body

Reply:
After  thinking about this for quite a while, I don't think there is another way I would do this.